_conferences/mblt-dev-2015.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "MBLTDev"
+website: http://mbltdev.ru/
```

Comment:
Wow, strange, there are a lot of speakers from different countries. 

Can you please add a screenshot? I'll send it to the organizers of the conference.